### DISCUSSION TITLE
Move Web UI translations to crowdin

### DISCUSSION THREAD

**Discussion by juanpicado** [verdaccio/verdaccio#2364] [Ideas]
Just recently the `docs/` and website were moved to this repository. I'd like to have the UI translations via crowdin 
https://github.com/verdaccio/verdaccio/tree/master/packages/plugins/ui-theme/i18n/translations
instead having them in the control system. This will make much easier for contributors.

**Comment by comradekingu** [comment]
Crowdin has no benefits for contributors and is a net detractor from getting contributions. I advice using Weblate instead.
